frontend/ui-pablo/app/components/Atoms/BigNumberInput.tsx

```diff
@@ -37,13 +37,17 @@ export const BigNumberInput: React.FC<BigNumberInputProps> = ({
 
   React.useEffect(() => {
     setter && setter(bignrValue);
+    // Only depends on setters, therefore we can omit dependencies.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [bignrValue]);
```

Comment:
@markostojanovic087 acc to react documentation setter are not created with each render https://reactjs.org/docs/hooks-reference.html#usestate

Reply:
Yes, but are you sure that you need the eslint rule exception for this case?
On mosaic I don't add the state setters as well, but I also don't need to add eslint rule exception. (But maybe you are using linter out of next),

Reply:
In this case, `eslint` can't understand if `setter` is a React state or not since it is coming in through the props. So, I think it is fine if we omit it. In Reality, though, it is suggested to use `useCallback` to not use eslint ignore.